Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py

```diff
@@ -106,12 +106,20 @@ def generic_command(client: MsGraphClient, args: Dict[str, Any]) -> CommandResul
         results = {'raw_response': response}
 
         if argToBoolean(args.get('populate_context', 'true')):
-            results['outputs'] = response.get('value')
+            results['outputs'] = get_response_outputs(response)
             results['outputs_prefix'] = 'MicrosoftGraph'
 
     return CommandResults(**results)  # type: ignore[arg-type]
 
 
+def get_response_outputs(response: dict):
+    if 'value' in response:
+        return response['value']
```

Comment:
dont we need to pop out `@odata.context` here as well ?

Reply:
no, since it's in the root, but not in the `value`.